algo/src/main/java/com/hartwig/serve/sources/ckb/curation/CkbCurator.java

```diff
@@ -31,45 +50,42 @@ public List<CkbEntry> run(@NotNull List<CkbEntry> ckbEntries) {
 
         for (CkbEntry ckbEntry : ckbEntries) {
             List<Variant> curatedVariants = Lists.newArrayList();
+            List<ClinicalTrial> curatedClinicalTrials = Lists.newArrayList();
             for (Variant variant : ckbEntry.variants()) {
-                curatedVariants.add(curate(variant));
+                curatedVariants.add(curateVariant(variant));
             }
-            curatedCkbEntries.add(ImmutableCkbEntry.builder().from(ckbEntry).variants(curatedVariants).build());
-        }
-
-        return curatedCkbEntries;
-    }
-
-    public void reportUnusedCurationEntries() {
-        int unusedEntryCount = 0;
-        for (CurationEntry entry : CurationFactory.VARIANT_MAPPINGS.keySet()) {
-            if (!evaluatedCurationEntries.contains(entry)) {
-                unusedEntryCount++;
-                LOGGER.warn(" Entry '{}' hasn't been used during CKB curation", entry);
+            for (ClinicalTrial clinicalTrial : ckbEntry.clinicalTrials()) {
+                List<Location> locations = Lists.newArrayList();
```

Comment:
Maybe rename to `curatedLocations` similar to how `curatedVariants` and `curatedClinicalTrials` are used?